Formula/sslscan.rb

```diff
@@ -17,10 +17,12 @@ class Sslscan < Formula
 
   def install
     system "make"
+    mkdir_p [bin, man1]
     system "make", "install", "PREFIX=#{prefix}"
   end
 
   test do
+    assert_match version.to_s, shell_output("#{bin}/sslscan --version")
```

Comment:
That addition was suggested by @ilovezfs . It's a quick simple test, and should be (at worst) harmless.